src/tracer.js

```diff
@@ -236,8 +243,13 @@ export default class Tracer {
             // reuse parent's baggage as we'll never change it
             ctx.baggage = parent.baggage;
 
-            parent.finalizeSampling();
```

Comment:
parent.finalized should behave same as before, I think, why do you remove it?

Reply:
Restored.